oz/redis_sessions/middleware.py

```diff
@@ -24,11 +24,18 @@ def _session_key(self):
 
             if not session_id:
                 session_id = oz.redis_sessions.random_hex(20)
-                self.set_secure_cookie(
-                    name="session_id",
-                    value=session_id.encode('utf-8'),
-                    domain=oz.settings.get("cookie_domain"),
-                    httponly=True)
+
+            session_time = oz.settings["session_time"]
+            kwargs = dict(
+                name="session_id",
+                value=session_id.encode('utf-8'),
+                domain=oz.settings.get("cookie_domain"),
+                httponly=True,
+            )
+            if session_time:
+                kwargs["expires_days"] = round(session_time/60/60/24)
+
+            self.set_secure_cookie(**kwargs)
```

Comment:
Wouldn't this set the secure cookie far more often than it should be? The `_cached_session_key` is scoped to a request, so I would think this would update the cookie for each request and constantly reset the expiration. I would think we do /not/ want to reset expiration time unless the session itself is reset.